superset-frontend/src/components/Table/Table.stories.tsx

```diff
@@ -593,19 +593,19 @@ const shoppingData: ShoppingData[] = [
   {
     key: 1,
     item: 'Floppy Disk 10 pack',
-    orderDate: Date.now(),
+    orderDate: new Date('2015-07-02T16:16:00Z').getTime(),
```

Comment:
Is there something akin to [freezegun](https://github.com/spulec/freezegun) for frontend code?

Reply:
Good question! I'll try to remember to look into it, but I'm not sure of one offhand. I just picked the timestamp of Superset's first commit since it's pretty arbitrary here. ¯\\\_(ツ)_/¯